deploy/swarm_deployment.md

```
@@ -0,0 +1,156 @@
+# Setup a remote server
+
+Install docker
+
+```
```

Markdownlint (0.37.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)